schema.sql

```diff
@@ -0,0 +1,74 @@
+CREATE TABLE music_albums (
+  id SERIAL PRIMARY KEY,
+  genre VARCHAR(255),
+  author VARCHAR(255),
+  source VARCHAR(255),
+  label VARCHAR(255),
+  publish_date INT,
+  archived BOOLEAN,
+  title VARCHAR(255),
+  artist VARCHAR(255),
+  on_spotify BOOLEAN
+);
+
+CREATE TABLE genres (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(255)
+);
+
+CREATE TABLE authors (
+  id SERIAL PRIMARY KEY,
+  first_name VARCHAR(255)
+  last_name VARCHAR(255)
+);
+
+CREATE TABLE Books (
+  item_id INT PRIMARY KEY,
+  publisher VARCHAR(255),
+  cover_state VARCHAR(50),
+  FOREIGN KEY (item_id) REFERENCES Item(id)
+);
+
+CREATE TABLE Label (
+   id INT PRIMARY KEY AUTO_INCREMENT,
+   title VARCHAR(255),
+   color VARCHAR(20)
+);
+
+CREATE TABLE Movies (
+    item_id INT PRIMARY KEY,
+    director VARCHAR(255),
+    release_date DATE,
+    silent BOOLEAN,
+    FOREIGN KEY (item_id) REFERENCES Item(id)
+);
+
+CREATE TABLE Games (
+    title VARCHAR(255),
+    item_id INT PRIMARY KEY,
+    publish_date DATE,
+    multiplayer BOOLEAN,
+    last_played_at DATE,
+    FOREIGN KEY (item_id) REFERENCES Item(id)
+);
+
+
+CREATE TABLE Sources (
+    id INT PRIMARY KEY AUTO_INCREMENT,
+    name VARCHAR(255),
+    url VARCHAR(255),
+    item_id INT,
+    FOREIGN KEY (item_id) REFERENCES Item(id)
+);
```

Comment:
- [ ] When testing the app everything is working well but when tried to list all the sources I have get the following error, kindly fix the issue:

<img width="654" alt="Screenshot 2023-09-23 at 4 30 58 PM" src="https://github.com/sergio-usma/Ruby-capstone-project/assets/53347966/27eb2531-b7e1-4ee0-8a62-7a73978acb29">

Reply:
well resolved! ✔️